src/ResourceManager/Resources/Commands.Resources/help/Set-AzureRmPolicyDefinition.md

```diff
@@ -82,7 +90,7 @@ Accept wildcard characters: False
 The credentials, account, tenant, and subscription used for communication with azure
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```

Comment:
@mentat9 would you mind running the `Update-MarkdownHelpModule` command again with the `-UseFullTypeName` parameter to preserve the fully qualified name in the types?